rateLimits/messageRateLimits.md

```diff
@@ -33,13 +33,14 @@ All Bandwidth messaging products are rate limited in some fashion. The default r
 | **Account**    | Inbound _API_ rate      | The rate that Bandwidth's API will accept new messages requests.                                                                                                                                            | 1 MPS                                                                                                                                                           |
 | **Per-Number** | Inbound _API_ rate      | The rate that Bandwidth's API will accept new messages requests from the same phone number                                                                                                                  | P2P - 1 MPS <br> A2P - ∞, up to account limit                                                                                                                   |
-| **Account**    | Burst _API_ rate        | The burst rate that Bandwidht's API will accept new messages. Essentially how quickly can the queue be filled. <br> <br> ⚠️ If queuing is **not** enabled, this will be same as the  **Inbound _API_ rate** | 1 MPS <br> -or- <br> **Account** Outbound _dequeue_ rate + 10 <br><br> Example: **Account** Outbound _dequeue_ rate is 5 MPS, the **Burst** rate will be 15 MPS |
+| **Account**    | Burst _API_ rate        | The burst rate that Bandwidth's API will accept new messages. Essentially how quickly can the queue be filled. <br> <br> ⚠️ If queuing is **not** enabled, this will be same as the  **Inbound _API_ rate** | 1 MPS <br> -or- <br> **Account** Outbound _dequeue_ rate + 10 <br><br> Example: **Account** Outbound _dequeue_ rate is 5 MPS, the **Burst** rate will be 15 MPS |
 
 ### 403 – LIMIT
-| Code                | Message                                                                                                     |
-|:--------------------|:------------------------------------------------------------------------------------------------------------|
-| message-rate-limit  | You can send ${0} messages per ${1} seconds calculated as the average over ${2} seconds. Your rate is: ${3} |
-| multi-message-limit | The number of messages [${0}] in the multi message request is greater than the limit [${1}].                |
+| Code                        | Message                                                                                                        |
+|:----------------------------|:---------------------------------------------------------------------------------------------------------------|
+| message-rate-limit-overall  | You can send ${0} messages per ${1} seconds calculated as the average over ${2} seconds. Your rate is: ${3}    |
```

Comment:
Did these error codes change? Looks like they are both return the error code "message-rate-limit".

Reply:
I changed them here https://github.com/Bandwidth/Development/pull/1683 but I haven't merged it yet

Reply:
Just realized I change the message but not the error code category in that PR. Need to fix that

Reply:
Just realized I change the message but not the error code category in that PR. Need to fix that

Reply:
Okay this is updated in the other PR now